index.html

```diff
@@ -190,17 +190,43 @@ <h2 class="testimonial-text">
     </section>
 
     <!-- Pricing -->
+    <!-- TODO: #27 Add bootstrap cards to pricing sections @hbrandon15-- -->
 
     <section id="pricing">
       <h2>A Plan for Every Dog's Needs</h2>
       <p>Simple and affordable price plans for your and your dog.</p>
+      
+      <div class="card">
+        <div class="card-header">Card Header</div>
+        <!-- card header -->
+        <h3>Chihuahua</h3>
+        <div class="card-body">Card Body</div>
+        <!-- card body -->
+        <h2>Free</h2>
+        <p>5 Matches Per Day</p>
```

Comment:
Bootstrap 5 documentation needed to be used. Cards were changed in the latest version.